File: AFBack/Features/Auth/DTOs/Response/TokenResponse.cs
namespace AFBack.Features.Auth.DTOs.Response;

public class TokenResponse
{
    public required string AccessToken { get; init; }
    public required string RefreshToken { get; init; }
    public required DateTime AccessTokenExpires { get; init; }
    public required DateTime RefreshTokenExpires { get; init; }
}

File: AFMobile/core/models/TokenResponse.ts
// core/models/TokenResponse.ts
// Delt DTO for token-respons fra auth-endepunkter.
// Tilsvarer TokenResponse i AFBack.
// Brukes av LoginResponseDTO og potensielt refresh-token-respons.

export interface TokenResponse {
  accessToken: string;
  refreshToken: string;
  accessTokenExpires: string;
  refreshTokenExpires: string;
}
